src/Features/CSharp/Portable/EncapsulateField/CSharpEncapsulateFieldService.cs

```diff
@@ -196,12 +193,13 @@ private static bool IsNew(IFieldSymbol field)
     private static string GenerateFieldName(string correspondingPropertyName)
         => char.ToLower(correspondingPropertyName[0]).ToString() + correspondingPropertyName[1..];
 
-    protected static string MakeUnique(string baseName, INamedTypeSymbol containingType)
+    private static string MakeUnique(string baseName, INamedTypeSymbol containingType)
     {
         var containingTypeMemberNames = containingType.GetAccessibleMembersInThisAndBaseTypes<ISymbol>(containingType).Select(m => m.Name);
         return NameGenerator.GenerateUniqueName(baseName, containingTypeMemberNames.ToSet(), StringComparer.Ordinal);
     }
 
-    protected override IEnumerable<SyntaxNode> GetConstructorNodes(INamedTypeSymbol containingType)
-        => containingType.Constructors.SelectMany(c => c.DeclaringSyntaxReferences.Select(d => d.GetSyntax()));
+    protected override IEnumerable<ConstructorDeclarationSyntax> GetConstructorNodes(INamedTypeSymbol containingType)
+        => containingType.Constructors.SelectMany(
+            c => c.DeclaringSyntaxReferences.Select(d => d.GetSyntax()).OfType<ConstructorDeclarationSyntax>());
```

Comment:
this is the core of hte fix.  we only want to update constructor bodies, not the entire type (which is what a primary constructor returns).